.github/workflows/build-golang.yml

```diff
@@ -14,39 +14,51 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os-artifact: [ [ubuntu-latest, linux], [windows-latest, windows], [macos-latest, macos] ]
+        os-artifact: [ [ubuntu-latest, linux], [windows-latest, windows-gnu], [macos-latest, macos] ]
```

Comment:
fix 3d. seems we're already [trying to do this in build_sdks.py](https://github.com/trinsic-id/okapi/blob/main/devops/build_sdks.py#L127), so the GitHub download workflow should match and we should download windows-gnu artifacts.  otherwise, there is no windows-gnu dir to copy from in the linked code.

Reply:
Good catch!